src/sentry/static/sentry/app/components/dashboards/widgetQueriesForm.tsx

```diff
@@ -162,6 +162,7 @@ class WidgetQueriesForm extends React.Component<Props> {
             flexibleControlStateSize
             stacked
             error={this.getFirstQueryError('orderby')?.orderby}
+            style={{marginBottom: space(1)}}
```

Comment:
Should this be put on the QueryWrapper instead?

Reply:
@markstory It was removed from `<QueryWrapper />` by @doralchan in https://github.com/getsentry/sentry/pull/24347 to remove the extra whitespace for other display types. This is an adjustment for the table display type.